### PR TITLE
Reference sourcemaps from GitHub.

### PR DESCRIPTION
**Pull request by cramforce**
This is better than local, but ideally this would point at the right tag.

### CODE REVIEW

**Comment by cramforce** [comment]
@erwinmombay Could we pass the tag as a flag to the compile step instead?

**Comment by erwinmombay** [comment]
@cramforce definitely doable, is that so we can get to the version folder? `w.a.o/ver/v0/${source}`?

**Comment by cramforce** [comment]
Nah, we point at Github just with the tag in the URL to get sources at the right version.

File: build-system/tasks/compile.js [hunk 1]
```diff
@@ -148,7 +148,8 @@ function compile(entryModuleFilename, outputDir,
         only_closure_dependencies: true,
         output_wrapper: wrapper,
         create_source_map: intermediateFilename + '.map',
-        source_map_location_mapping: '|http://localhost:8000/',
+        source_map_location_mapping:
+            '|https://raw.githubusercontent.com/ampproject/amphtml/master/',
```
Comment (erwinmombay) [inline build-system/tasks/compile.js]: so if we had the `tag` should be something like https://rawgit.com/ampproject/amphtml/1454976295735/src/

ex: https://rawgit.com/ampproject/amphtml/1454976295735/src/3p.js

**Comment by jridgewell** [comment]
Why point to `raw.github.com` instead of the Github source code viewer?

**Comment by erwinmombay** [comment]
yeah sorry, i meant `https://raw.githubusercontent.com/ampproject/amphtml/1454976295735/src/`

`https://raw.githubusercontent.com/ampproject/amphtml/1454976295735/src/3p.js`

**Comment by erwinmombay** [comment]
@cramforce so since the future tag is going to be `$internalRuntimeVersion` couldn't we just use that?

**Comment by cramforce** [comment]
@erwinmombay yep!

**Comment by cramforce** [comment]
@jridgewell Not sure what you mean? We need the raw source.

**Comment by erwinmombay** [comment]
@cramforce LGTM pending, the tag version change to the path

**Comment by jridgewell** [comment]
Never mind my comment.

**Comment by cramforce** [comment]
Arghh, it works with the tag, but we are missing all the build artifacts. The right solution is to embed the source file into the map as a data-uri.

**Comment by cramforce** [comment]
But, it is still pretty nice step forward, so lets roll with it.

**Pull request merged**